libs/ci/github/github.go

```diff
@@ -471,6 +478,7 @@ func ConvertGithubPullRequestEventToJobs(payload *github.PullRequestEvent, impac
 				RequestedBy:        *payload.Sender.Login,
 				CommandEnvProvider: CommandEnvProvider,
 				StateEnvProvider:   StateEnvProvider,
+				SkipMergeCheck: 	skipMerge,
```

Comment:
I think you should also added it in the rest of the branch tree, any line where it says jobs = append() should also have the line for SkipMergeCheck field

Reply:
Wen through and added the logic plus the option to all locations where a Job struct is created.